AutocompleteClient/FW/UI/CIOAutocompleteViewController.swift

```diff
@@ -233,6 +233,11 @@ public class CIOAutocompleteViewController: UIViewController {
         }
         
         self.viewModel.set(searchResult: autocompleteResult) { [weak self] in
+            if let searchTerm = self?.viewModel.searchTerm,
+                let resultCount = self?.viewModel.results.reduce(0, { (res, nextSection) -> Int in return res + nextSection.items.count }){
+                self?.constructorIO.tracking.trackResultsLoaded(searchTerm: searchTerm, resultCount: resultCount)
+            }
```

Comment:
It is confusing, but `trackSearchResultsLoaded` will be called by the consuming code when a page of actual search results loads and not automatically when autocomplete results are returned.  So these lines should be removed and ideally we would have a button on the library demo page below the clapper that people can press as an example.

Reply:
<img width="501" alt="screen shot 2018-07-03 at 11 26 10 am" src="https://user-images.githubusercontent.com/524424/42214846-86fb679e-7eb4-11e8-85ef-d37f0aafeba7.png">